### DISCUSSION TITLE
1.2.1 Screen not refreshing

### DISCUSSION THREAD

**Discussion by imtrobin** [rustdesk/rustdesk#5216] [General]
### Bug Description

Hi, I connect using 1.2.1, the client screen often freezes without changes, I have to reconnect to get screen updates.

### How to Reproduce

1. Connect
2. Leave it idle , it can be 2 min or longer. There is no screen saver on both sides
3. Client screen freezes with not updates

### Expected Behavior

screen continue to send update

### Operating system(s) on local side and remote side

win10

### RustDesk Version(s) on local side and remote side

1.2.1

### Screenshots

none

**Comment by rustdesk** [comment]
Move to discussions, because nothing we can help based on your information for the time being, too specicial case.

**Reply by imtrobin** [reply]
What kind of info do u need?
